src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -261,12 +261,16 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
         }
 
         private static bool BackendSupportsCustomCertificateHandling =>
-            RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ||
+#if unix
```

Comment:
Actually, I just discovered that by leaving this line in:

```c#
RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ||
```

it will avoid calling into the p/invoke since the expression will be true.  So, yes, I can remove the 'if' def logic.